build-coatjava.sh

```diff
@@ -60,6 +63,7 @@ done
 
 src_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 prefix_dir=$src_dir/coatjava
+clara_home=$(dirname $prefix_dir)/clara-home
```

Comment:
why a relative path and not an absolute path?
```suggestion
clara_home=$prefix_dir/clara-home
```

Reply:
wouldn't that put it inside coatjava?

Reply:
yes... this confused me; how about this for clarity:
```bash
clara_home=$src_dir/clara-home
```